src/inverted-index.js

```diff
@@ -57,26 +63,28 @@ class InvertedIndex {
     return index;
   }
 
+  /**
+   * @return{Object} index - That maps words to locations(documents)
+   */
   getIndex() {
     return this.index;
   }
 
   /**
-   * @param{String} searchString, searchIndex - Search query
+   * @param{String} searchWords - Search query
+   * @param{String} indexToSearch - Index to query
    * @return{Object} searchResults - Maps searched words to document locations
    */
   searchIndex(searchWords, indexToSearch) {
     const searchResults = {};
     const searchTerms = this.uniqueWords(searchWords);
-    for (let word of searchTerms) {
+    searchTerms.forEach((word) => {
       if (indexToSearch[word]) {
         searchResults[word] = indexToSearch[word];
       } else {
         searchResults[word] = `We are Sorry but ${word} is not found in our database`;
```

Comment:
What do you think about breaking this line as such:

> searchResults[word] =
          We are Sorry but ${word} is not found in our database;